cogs/onmyoji.py

```diff
@@ -95,6 +101,59 @@ def generate_user_shikigami_locations(cls, shiki_name):
                 user_database_filtered_locations.append(temp_list)
         return user_database_filtered_locations
 
+class Embeds:
+    def shard_trading_embed(self, user):
+        need_list = ''
+        have_list = ''
+        for i in self.shard_trading_db[str(user.id)]['need']:
+            number, shiki = self.shard_split_variable(i)
+            need_list = f"{need_list}{bold(number)} {shiki}\n"
```

Comment:
This pattern seems odd, its not obvious that you're concatenating to `need_list`
```suggestion
            need_list += f"{bold(number)} {shiki}\n"
```